deno/lib/types.ts

```diff
@@ -903,6 +918,9 @@ export class ZodString extends ZodType<string, ZodStringDef> {
   get isCUID2() {
     return !!this._def.checks.find((ch) => ch.kind === "cuid2");
   }
+  get isULID() {
+    return !!this._def.checks.find((ch) => ch.kind === "cuid2");
```

Comment:
probably a copy-paste mistake
```suggestion
    return !!this._def.checks.find((ch) => ch.kind === "ulid");
```

Reply:
good catch, addressed!